frontend/catalyst/__init__.py

```diff
@@ -81,15 +81,15 @@
     "mlir_quantum._mlir_libs._quantumDialects.mitigation"
 )
 
-from catalyst import debug, logging
+from catalyst import debug, logging, passes
 from catalyst.api_extensions import *
 from catalyst.api_extensions import __all__ as _api_extension_list
 from catalyst.autograph import *
 from catalyst.autograph import __all__ as _autograph_functions
 from catalyst.compiler import CompileOptions
 from catalyst.debug.assertion import debug_assert
 from catalyst.jit import QJIT, qjit
-from catalyst.passes import Pass, PassPlugin, apply_pass, apply_pass_plugin, pipeline
+from catalyst.passes import pipeline
```

Comment:
Unfortunately the `pipeline` function was already exported top-level last release. @josh146 should leave it as for continuity or should we remove it now (small breaking change)?

Reply:
As discussed this commit makes it so it's only exported into the top-level namespace, without appearing in the `passes` namespace: https://github.com/PennyLaneAI/catalyst/pull/1430/commits/21798dcc4feeae8c7b3f0e7662b7b5a1539e8112